### ISSUE TITLE
compatibility with Samsung Tool PRO

### ISSUE THREAD

**Issue by RME-Systems** [nvaccess/nvda#13188]
### Is your feature request related to a problem? Please describe.

I currently work in cell phone repair and use a software: samsung tool pro.
https://z3x-team.com/products/samsung-activation/

the problem is that I cannot access parts of the software in any way (arrows, tab, OCR, object browser, or screen review).
I can only access using JAWS OCR.
the parts that I can't access are: some tabs at the top which are: model, flash, backup, repair, unlock, settings, history, downloads

### Describe the solution you'd like

I would like that if possible compatibility can be added for the mentioned software maybe with keyboard shortcuts like: unlock ctrl + u repair ctrl + r, etc.
### Additional context

I tried to contact the z3x team, but they couldn't understand me.
Anything I am available, thank you

**Comment by OzancanKaratas** [comment]
If a software is not usable with any screen reader, the problem is that the software does not meet accessibility standards. Samsung tends not to use accessibility standards. There is nothing NVDA developers can do about it.

@josephsl, Please give your opinion.

**Comment by ABuffEr** [comment]
> I can only access using JAWS OCR.

And this, indeed, is the only oddity. What happens if you try with NVDA/Windows OCR? Maybe you have to set the correct object to start recognition... Have you tried with [Tesseract OCR?](https://github.com/lukaszgo1/nvda-ocr)
Looking at main .exe folder, do you see files related to QT4 (often used in similar software)?
As it already said, anyway, there is nothing NVDA devs can do here, so this issue will be closed soon, I think.

**Comment by Brian1Gaff** [comment]
Yes its time for folk to vote with their feet or at the very least campaign 
that software should be fixed when its inaccessible in my view. It seems to 
work, albeit slowly. I suspect until every software writer understands the 
ethos of accessible software, nothing will actually change and so saying 
nothing is not an option. I have found that in the past trying to get work 
around at the add on and script level is a recipe for disaster, since the 
next version will undoubtedly be totally different and need a whole set of 
new solutions.
 In my view the minimum standards should be it works with the inbuilt 
screenreader in an OS but many packages simply don't. Call centre software 
is often the worst.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by Adriani90** [comment]
@RME-Systems are you still having this issue with NVDA 2024.4.1 and last version of Samsung tool pro?

**Comment by RME-Systems** [comment]
@Adriani90 Hi, yes. I still have the problem.

**Comment by seanbudd** [comment]
Is this still an issue in 2026.1 or newer? Can you please open a bug report instead of a feature request, filling out all the details including a log file?
